.github/CODEOWNERS

```diff
@@ -50,10 +50,12 @@
 # Fluid Devtools source
 /packages/tools/devtools/**/src                @microsoft/fluid-cr-devtools
 
-# Non-internal API changes
-/**/api-report/*.public.api.md                 @microsoft/fluid-cr-api
-/**/api-report/*.beta.api.md                   @microsoft/fluid-cr-api
-/**/api-report/*.alpha.api.md                  @microsoft/fluid-cr-api
+# API report changes
+# TODO: if we ever add `.internal` API reports, we will want to omit them here
+/**/api-report/*.api.md                        @microsoft/fluid-cr-api
+/build-tools/**/api-report/*.api.md            @microsoft/fluid-cr-infra
+/server/**/api-report/*.api.md                 @microsoft/fluid-cr-server
+/tools/**/api-report/*.api.md                  # Do not require API review for tools packages
```

Comment:
@Josmithr  we shouldn't add ownership to groups unless they explicitly opt into it, so adding cr-infra, and cr-server shouldn't be done unless those groups know and agree to the requirements they are signing up for, to review all changes. it might be a better change to just limit cr-api to the packages directory, rather than creating new layers of ownership

Reply:
i would expect some members of those teams to sign off here, to signify that they know and agree to the requirements.

Reply:
#22294